Dockerfile

```diff
@@ -1,5 +1,5 @@
 # syntax = docker/dockerfile:1.4
-FROM python:3.10 AS compile-stage
+FROM python:3.13.0a4-slim AS compile-stage
 RUN --mount=type=cache,target=/var/cache/apt \
     apt update && \
     apt install -y --no-install-recommends \
```

Comment:
You should to add libraries for the compilation of Pillow 9.5 for Python 3.12

```diff
 RUN --mount=type=cache,target=/var/cache/apt \
     apt update && \
     apt install -y --no-install-recommends \
         build-essential gcc neovim fish less iputils-ping postgresql-client \
+        zlib1g-dev libjpeg-dev libraqm-dev libimagequant-dev libtiff-dev \
+        libfreetype6-dev liblcms2-dev libwebp-dev libharfbuzz-dev libfribidi-dev \
+        libtiff5-dev \
+        libwebp-dev \
         ack
```

without these libs, I can't compile the Docker image